etc/docker-compose.yml

```diff
@@ -1,39 +1,63 @@
-version: '3'
+version: '3.9'
+name: 'reth'
 
 services:
+  reth:
+    restart: unless-stopped
+    image: ghcr.io/paradigmxyz/reth
+    ports:
+      - '9000:9000'
```

Comment:
what do we think about 8545? 30303? 8551 for CL?

Reply:
depends on what we want the default to be. Maybe expose everything thus making it easier for people to just comment the ports they want to close? 

also fyi exposing some of these ports is useless unless the http rpc is active, so maybe we should enable it by default?